storybooknative/src/CenterView/index.js

```diff
@@ -5,17 +5,18 @@ import { View } from "react-native";
 
 import styled from "../styled";
 
-const Main = styled(View, {
+const Main = styled(View, props => ({
   flexGrow: 1,
   justifyContent: "center",
-  alignItems: "center",
+  alignItems: props.align || "center",
   backgroundColor: "#f5fcff",
-});
+}));
 
 type Props = {|
   children: React.ChildrenArray<React.Node> | null,
+  align: "flex-start" | "center" | "flex-end" | "stretch",
```

Comment:
Props should be marked readonly `+align: "flex-start" | "center" | "flex-end" | "stretch"`

Also, maybe we should extract this align-enum somewhere and export it. It could be used by other components in the future also?